frontend/src/screens/ConnectAlbyAccount.tsx

```diff
@@ -34,7 +34,7 @@ export function ConnectAlbyAccount({ connectUrl }: ConnectAlbyAccountProps) {
         <div className="grid grid-cols-1 md:grid-cols-2 w-full gap-2 mt-5">
           <Card className="w-full relative">
             <CardHeader className="flex flex-col justify-center items-center text-center p-4">
-              <ZapIcon className="w-6 h-6" />
+              <ZapIcon className="size-6" />
```

Comment:
The cards are really big now and the content doesn't fit on the screen, not sure why:

<img width="1029" height="1273" alt="image" src="https://github.com/user-attachments/assets/924f5ea8-b66a-4244-af47-94ba94ecfc46" />